res/xml/preferences.xml

```diff
@@ -152,6 +152,13 @@
                     android:persistent="true"
                     android:defaultValue="true"/>
 
+            <CheckBoxPreference
+                    android:key="sync_read_flag"
+                    android:title="@string/ui_sync_read_flag_label"
+                    android:summary="@string/ui_sync_read_flag_desc"
+                    android:persistent="true"
+                    android:defaultValue="false"/>
+
```

Comment:
Only for SMS and MMS, see https://github.com/jberkel/sms-backup-plus/pull/508/files#diff-eb2588bd89d5369abcf67438266213a8R119.